neutone_sdk/utils.py

```diff
@@ -165,14 +177,17 @@ def save_neutone_model(
         loaded_model.reset()
         loaded_model.is_resampling()
 
-        log.info("Testing offline mode...")
-        for input_sample in get_default_audio_samples():
-            offline_sr = input_sample.sr
-            offline_bs = 4096
-            loaded_model.set_daw_sample_rate_and_buffer_size(offline_sr, offline_bs)
-            offline_params = tr.rand((MAX_N_PARAMS, input_sample.audio.size(1)))
-            offline_rendered_sample = loaded_model.forward_offline(input_sample.audio, offline_params)
-            break  # Only test one sample to reduce export time
+        if test_offline_mode:
+            log.info("Testing offline mode...")
+            for input_sample in get_default_audio_samples():
+                offline_sr = input_sample.sr
+                offline_bs = 4096
+                loaded_model.set_daw_sample_rate_and_buffer_size(offline_sr, offline_bs)
+                offline_params = tr.rand((MAX_N_PARAMS, input_sample.audio.size(1)))
+                offline_rendered_sample = loaded_model.forward_offline(
```

Comment:
Maybe we can add a check here that rendering offline vs online yields the same result

Reply:
This might be non-trivial due to the buffering delay, added a TODO for now.